src/pages/updates/work-in-progress/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Work in progress
+description:
+  This page provides early visibility for new Carbon concepts and current work
+  in progress.
+---
+
+<PageDescription>
+
+An opportunity to see early Carbon concepts and ideas, and experiment with work
+in progress.
+
+</PageDescription>
+
+<InlineNotification>
+  This work in progress is not actively supported by the Carbon team, and is not
+  approved for production use. We gratefully accept feedback or comment.
+</InlineNotification>
+
+## Introduction
+
+The Carbon team is always exploring different ways to serve our users. We work
+in the open, and this page is here to give you early previews of what we're
+working on.
+
+That could mean making an unfinished Sketch kit available for download, linking
+to GitHub issues for proposed components and patterns, or sharing graphics files
+of concepts for how we might present content.
+
+Not everything here may make it into production, while other pieces might become
+central pieces of the design system. It's an early opportunity for you to
+provide feedback and have a say in Carbon's direction.
+
+### Tools and resources
+
+<CardGroup>
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'GC8C8OMU', 0)}
+    title="Carbon mobile sketch kit"
+    href="https://www.sketch.com/s/a3343128-adb6-489c-9e62-709d89ba76e9"
+    actionIcon="launch"
+  />
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'GP38VQMY', 0)}
+    title="Data viz sticker sheet"
+    href="https://ibm.box.com/s/epx74uadotxbx36aknn1s2nmvhpzgk4o"
+    actionIcon="download"
+  />
+</CardGroup>
+
+### Components and patterns
+
+<CardGroup>
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'B3AN6GVA', 0)}
+    title="Tree view component"
+    href="https://carbon-website-git-fork-designertyler-treeview03092020.carbon-design-system.now.sh/components/tree-view/usage"
+  />
+</CardGroup>
+
+### Website design and content
+
+<CardGroup>
+  <MiniCard
+    onClick={() => fathom('trackGoal', '1WVDJ6KO', 0)}
+    title="Community index"
+    href="https://www.carbondesignsystem.com/community/component-index/"
+  />
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'C9ESE7PV', 0)}
+    title="IA exploration"
+    href="https://github.com/carbon-design-system/carbon-website/issues/1516"
+  />
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'ZASURK2Q', 0)}
+    title="Activation concepts"
```

Comment:
It seems a little like oversharing to share this whole file. There no real reason for people to have the asset. I wonder if we could just drop these as images into a github issue and share that instead.

Reply:
I think that's the right solution, yes. We probably should have a github issue outlining some of these concepts anyway, and then be able to link to that. Would give some context to the images, too.

In the absence of the github issue, I'm ok with linking to the file, but the issue is the best longer term solution.

Reply:
I definitely agree with the approach of using issues for context, rather than files. Issues provide an opportunity for comments too.

Reply:
I created #1536 as an issue - which needs populating by @janchild / @jeanservaas with a little more info. Then I can update the link from the WIP page to link to that issue rather than a Sketch file.